src/vs/workbench/api/browser/mainThreadInteractiveSession.ts

```diff
@@ -28,8 +24,7 @@ export class MainThreadInteractiveSession implements MainThreadInteractiveSessio
 		extHostContext: IExtHostContext,
 		@IInteractiveSessionService private readonly _interactiveSessionService: IInteractiveSessionService,
 		@IInteractiveSessionContributionService private readonly interactiveSessionContribService: IInteractiveSessionContributionService,
-		@IExtensionsWorkbenchService private readonly extensionsWorkbenchService: IExtensionsWorkbenchService,
-		@ILogService private readonly logService: ILogService,
+		// @ILogService private readonly logService: ILogService,
```

Comment:
commented out code?

Reply:
I'd rather just leave it or now since I use it for testing a lot